### ISSUE TITLE
persistenceAdapter not configured

### ISSUE THREAD

**Issue by zipavlin** [LokiJS-Forge/LokiDB#114]
## I'm submitting a...
<pre><code>[x] Bug report</code></pre>

## Current behavior
TL;DR: Unable to use persistence.
I am trying to manually load and save database, but I always get either error 'persistenceAdapter not configured' or no error but also no persistence. Lokijs on the other hand works with default options, but I need fulltext search. Is it possible that behavior/options changed since moving from lokisj?

## Expected behavior
To be able to use persistence (loading and saving database from disk).

## Minimal reproduction of the problem with instructions
*STEPS TO REPRODUCE*
```
const { Loki } = require('@lokidb/loki');
const loki  = new Loki('database.db');
loki.loadDatabase();
```

## What did I try?
- default options
- with _options.env: 'NODEJS'_
- with _options.adapter: new FSStorage()_ (const { FSStorage } = require('@lokidb/fs-storage');)
  - with and without _FSStorage.register()_
- with _options.autoload: true_ and _options.autoloadCallback_ (callback is not called)
- with _options.autosave_ and _options.autosaveInterval: 1000_

## Environment
<pre><code>
LokiDB version: 2.0.0-beta.6
Browser/Node version: NODE 8.9.4
</code></pre>

Guys, this project is great and I cannot wait for documentation :) Thanks for your hard work!

## EDIT:
I managed to get something working :+1: ( I found [documentation](https://lokijs-forge.github.io/LokiDB/api/index.html) - and I noticed the sidebar :))
It seems lokidb needs to call:
```
database.initializePersistence({
  autoload: true
});
```

**Comment by zipavlin** [comment]
@Viatorus Thank you!

I unded up using
```
const { FSStorage } = require('@lokidb/fs-storage');
FSStorage.register();
```
with custom load proccess that uses
```
await this.database.initializePersistence({
  autoload: fs.existsSync(this.databaseFile) // autoload only if file exists
});
// check if collections exist -> create and populate them if not
```
